app/models/medium.rb

```diff
@@ -1109,10 +1091,10 @@ def valid_annotations_status?
 
   private
 
-    # media of type kaviar associated to a lesson and script do not require
+    # media of type lesson_material associated to a lesson and script do not require
```

Comment:
In other comments, we use the CamelCase versions...

Reply:
Fixed.